tools/wpt/run.py

```diff
@@ -1,3 +1,4 @@
+# wpt command:run:main
```

Comment:
I'm having trouble understanding the purpose of this comment. Is it a vestige of some previous implementation strategy? Or am I missing something about the current patch?